bench/syclblas/syclblas_benchmark_level1.cpp

```diff
@@ -45,10 +45,11 @@ BENCHMARK(scal, syclblas_level_1) {
 
   auto in = ex.template allocate<ScalarT>(size);
   ex.copy_to_device(v1.data(), in, size);
-  benchmark<>::flops_units_t flops =
-      benchmark<>::measure(reps, size * 1, [&]() {
+  benchmark<>::datapoint_t flops =
+      benchmark<>::measure(reps, size * 1, [&]() -> cl::sycl::event {
```

Comment:
why the explicit return type?

Reply:
It's not required, but it produces errors more clearly if we forget to return an event.